sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkGetColumnsOperation.scala

```diff
@@ -131,7 +131,7 @@ private[hive] class SparkGetColumnsOperation(
    */
   private def getColumnSize(typ: DataType): Option[Int] = typ match {
     case dt @ (BooleanType | _: NumericType | DateType | TimestampType |
-               CalendarIntervalType | NullType) =>
+               CalendarIntervalType | NullType | YearMonthIntervalType | DayTimeIntervalType) =>
```

Comment:
Size of what does it return? `CalendarIntervalType`, `YearMonthIntervalType`, `DayTimeIntervalType` are returned as strings in rowSets.

Reply:
`CalendarIntervalType` return it's `defaultSize` (4 + 4 + 8 = 16).
It seems `YearMonthIntervalType` and `DayTimeIntervalType` should return `defaultSize` too.